tests/setup.ts

```diff
@@ -23,10 +26,20 @@ export const setup = async () => {
     process.env['NANGO_DB_NAME'] = 'postgres';
     process.env['NANGO_DB_MIGRATION_FOLDER'] = './packages/shared/lib/db/migrations';
     process.env['TELEMETRY'] = 'false';
-};
+}
+
+export async function setup() {
+    await Promise.all([setupPostgres()]);
```

Comment:
promise.All of one promise?

Reply:
There will be ElasticSearch soon enough (It's just a PR split from my big PR)